tests/integration/hive_metastore/test_migrate.py

```diff
@@ -30,14 +30,14 @@
 
 
 @retried(on=[NotFound], timeout=timedelta(minutes=2))
-def test_migrate_managed_tables(ws, sql_backend, runtime_ctx, make_catalog, make_schema):
+def test_migrate_managed_tables(ws, sql_backend, runtime_ctx, make_catalog):
     if not ws.config.is_azure:
         pytest.skip("temporary: only works in azure test env")
-    src_schema = make_schema(catalog_name="hive_metastore")
+    src_schema = runtime_ctx.make_schema(catalog_name="hive_metastore")
```

Comment:
@nkvuong : When should the `runtime_ctx.make_schema` be used and when should the `make_schema` fixture be used?